### PR TITLE
fail on missed entries

### LINKED ISSUE #213: [Proposal] fall if entry not found
Hi!

I test `./<some-path>/someFileName.js` file with size-limit.

Path to file or file's name can be changed. If I don't change `.size-limit.json` I got a warning `File not found: ./<some-path>/someFileName.js`.

Maybe should add a flag like `--no-warnings` and exit size-limit with error code in this cases?

I can implement it if you're agree.

### PR DESCRIPTION
**Pull request by binjospookie**
Close #213

### CODE REVIEW

File: packages/size-limit/create-reporter.js [hunk 1]
```diff
@@ -167,7 +168,8 @@ function createHumanReporter (process, isSilentMode = false) {
       }
 
       if (config.failed) {
-        let fix = 'Try to reduce size or increase limit'
+        let fix =
+          'Try to reduce size, increase limit or specify correct file path'
```
Comment (ai) [inline packages/size-limit/create-reporter.js:172]: The missed path is a rare case. We should show `specify corrent file path` only if we had this type of error (we can add `config.missed`)

Comment (binjospookie) [reply packages/size-limit/create-reporter.js:172]: You're right

**Comment by ai** [comment]
Released in 4.10.1

**Comment by binjospookie** [comment]
Thank you

**Pull request merged**